src/events.cpp

```diff
@@ -1380,7 +1380,7 @@ void onInventoryUpdate(Player* player, Item* item, slots_t slot, bool equip)
 	scriptInterface.callVoidFunction(4);
 }
 
-void onNetworkMessage(Player* player, uint8_t recvByte, NetworkMessage* msg)
+void onNetworkMessage(Player* player, uint8_t recvByte, NetworkMessage_ptr msg)
```

Comment:
```cpp
void onNetworkMessage(Player* player, uint8_t recvByte, const NetworkMessage_ptr& msg)
```

Reply:
Cannot be passed as const reference because we release the pointer in the end.

Reply:
Yes, I just noticed, using const was not the right option, although you can still use non -constant references in this method